ci-operator/config/openshift/tektoncd-triggers/openshift-tektoncd-triggers-release-next.yaml

```diff
@@ -0,0 +1,53 @@
+tag_specification:
+  name: '4.2'
+  namespace: ocp
+promotion:
+  namespace: openshift
+  name: tektoncd-next
+base_images:
+  base:
+    name: '4.2'
+    namespace: ocp
+    tag: base
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+canonical_go_repository: github.com/tektoncd/triggers
+binary_build_commands: make install
+images:
+- dockerfile_path: openshift/ci-operator/knative-images/controller/Dockerfile
```

Comment:
you didnt have to keep the `knative-images` name but i guess it's a pain to change,